src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -165,13 +165,25 @@ async function makeInstallScript(debuggerPath: string, isWindows: boolean): Prom
     const delvePath: string = path.posix.join(debuggerPath, 'dlv')
     const installOptions: SpawnOptions = { env: { ...process.env } }
     let delveVersion: string = ''
+    installOptions.env!['GO111MODULE'] = 'off'
 
     // It's fine if we can't get the latest Delve version, the Toolkit will use the last built one instead
     try {
-        const goPath: string = JSON.parse(execSync('go env -json').toString()).GOPATH
+        const goPath: string = JSON.parse(execFileSync('go', ['env', '-json']).toString()).GOPATH
         let repoPath: string = path.join(goPath, 'src', DELVE_REPO)
 
-        if (!pathExistsSync(repoPath)) {
+        function getDelveVersion(repo: string, silent: boolean): string {
```

Comment:
I don't mind inline functions (they are self documenting) but do we have any prior art for this in VSC?

Reply:
we have a few